core/templates/core/merge_report.html

```diff
@@ -0,0 +1,15 @@
+{% extends "core/emlo_logined.html" %}
+
+{% block content %}
+    <h1> Merge -- Report </h1>
+    <h1>{{ name }}</h1>
+    <span>Following relation moved to selected records:</span>
```

Comment:
"Following relation moved to selected records:" -> "The following related records were merged:"